src/main/java/cuchaz/enigma/translation/mapping/serde/TinyMappingsWriter.java

```diff
@@ -0,0 +1,115 @@
+package cuchaz.enigma.translation.mapping.serde;
+
+import com.google.common.collect.Lists;
+import cuchaz.enigma.ProgressListener;
+import cuchaz.enigma.translation.MappingTranslator;
+import cuchaz.enigma.translation.Translator;
+import cuchaz.enigma.translation.mapping.EntryMapping;
+import cuchaz.enigma.translation.mapping.MappingDelta;
+import cuchaz.enigma.translation.mapping.VoidEntryResolver;
+import cuchaz.enigma.translation.mapping.tree.EntryTree;
+import cuchaz.enigma.translation.mapping.tree.EntryTreeNode;
+import cuchaz.enigma.translation.representation.entry.ClassEntry;
+import cuchaz.enigma.translation.representation.entry.Entry;
+import cuchaz.enigma.translation.representation.entry.FieldEntry;
+import cuchaz.enigma.translation.representation.entry.MethodEntry;
+import cuchaz.enigma.utils.Utils;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.io.Writer;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Comparator;
+import java.util.HashSet;
+import java.util.Set;
+
+public class TinyMappingsWriter implements MappingsWriter {
+	private static final String VERSION_CONSTANT = "v1";
+
+	// HACK: as of enigma 0.13.1, some fields seem to appear duplicated?
+	private final Set<String> writtenLines = new HashSet<>();
+	private final String nameObf;
+	private final String nameDeobf;
+
+	public TinyMappingsWriter(String nameObf, String nameDeobf) {
+		this.nameObf = nameObf;
+		this.nameDeobf = nameDeobf;
+	}
+
+	@Override
+	public void write(EntryTree<EntryMapping> mappings, MappingDelta<EntryMapping> delta, Path path, ProgressListener progress) {
+		try {
+			Files.deleteIfExists(path);
+			Files.createFile(path);
+		} catch (IOException e) {
+			e.printStackTrace();
```

Comment:
I believe these should rather throw the exception

Reply:
The other writers just print the stacktrace. Might batch change these a bit later.